releasenotes/notes/use_local_defined_servers_include_openntpd_support-73c99224234b225f.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Include `/etc/ntpd.conf` and `/etc/openntpd/ntpd.conf` for `use_local_defined_servers`
```

```suggestion
    NTP check: Include `/etc/ntpd.conf` and `/etc/openntpd/ntpd.conf` for `use_local_defined_servers`
```

---

I _think_ monospace requires double backticks around the text
```suggestion
    NTP check: Include ``/etc/ntpd.conf`` and ``/etc/openntpd/ntpd.conf`` for ``use_local_defined_servers``.
```

---

Yes, good catch!